jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ErrorPageErrorHandler.java

```diff
@@ -75,7 +75,19 @@ public String getErrorPage(HttpServletRequest request)
             if (errorPage != null)
                 matchedThrowable = exClass;
 
-            th = (th instanceof ServletException) ? ((ServletException)th).getRootCause() : null;
+            if (th instanceof ServletException)
```

Comment:
If we change the ERROR_EXCEPTION here, then the LOG.DEBUG generated down at line 141/142 will be wrong.

Also, presumably jetty has had this behaviour for a very long time, so I'm wary about changing it ... @gregw WDYT?

Reply:
I think we need to provide a compliance mode to at least be able to switch back to the old behaviour, as we have offered this way for some time.